### PR TITLE
Changed Titlebar Color of Window.

### PR DESCRIPTION
**Pull request by J0nathan550**
Changed Titlebar color of the Window to pure black. Better then white bright color that doesn't match the style of the launcher.

Works both on Windows and Linux.
Linux Mint  Работает  - Oracle VM VirtualBox 02 03 2024 18_54_42
SIT Manager 02 03 2024 18_51_32

### CODE REVIEW

File: SIT.Manager.Avalonia/Views/MainWindow.axaml.cs [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+        TitleBar.InactiveBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.InactiveForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.ButtonForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonHoverBackgroundColor = new Color(0xFF, 0x11, 0x11, 0x11);
+        TitleBar.ButtonHoverForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonPressedBackgroundColor = new Color(0xFF, 0x21, 0x21, 0x21);
+        TitleBar.ButtonPressedForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
+        TitleBar.ButtonInactiveBackgroundColor = new Color(0xFF, 0x00, 0x00, 0x00);
+        TitleBar.ButtonInactiveForegroundColor = new Color(0xFF, 0xFF, 0xFF, 0xFF);
```
Comment (Plootie) [inline SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: Are these exposed in the xaml at all? I'd rather use xaml for this if possible

Comment (J0nathan550) [reply SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: Let me see...

Comment (J0nathan550) [reply SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: It is, but it's not recognizable by the intellisense. Let me try to fix that.

Comment (J0nathan550) [reply SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: No, just tested it doesn't want to grab those values from xaml. :( we have to use it only through source.

**Comment by J0nathan550** [comment]
Maybe, I am blind, here is the documentation.
https://amwx.github.io/FluentAvaloniaDocs/pages/Controls/AppWindow#related-enumsclasses

File: SIT.Manager.Avalonia/Views/MainWindow.axaml.cs (hunk 1 above)
Comment (J0nathan550) [reply SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: Maybe, I am blind, here is the documentation.
https://amwx.github.io/FluentAvaloniaDocs/pages/Controls/AppWindow#related-enumsclasses

Comment (J0nathan550) [reply SIT.Manager.Avalonia/Views/MainWindow.axaml.cs:24]: No chance, by the way UWP has done it same way of accessing color elements. You cannot change them in xaml file, but you only can change them in initialization of the source code. So bad luck on this one.

**Comment by J0nathan550** [comment]
@Plootie, there no way to access those items through xaml. Do you wish to create just function with name "Create Titlebar" or else?

**Comment by artehe** [comment]
> @Plootie, there no way to access those items through xaml. Do you wish to create just function with name "Create Titlebar" or else?

Ye just putting them into a function called CreateTitlebar would work for me, and I think Plootie would agree.

File: SIT.Manager.Avalonia/App.axaml [hunk 2]
```diff
@@ -23,7 +24,7 @@
 	</Application.Resources>
 	
     <Application.Styles>
-		<sty:FluentAvaloniaTheme />
+		<sty:FluentAvaloniaTheme CustomAccentColor="#323232" />
```
Comment (artehe) [inline SIT.Manager.Avalonia/App.axaml:27]: Do we really want to change this..... Makes some things really hard to see..... I'm not against the color being changed just not convinced a dark grey is the right choice.

With this change
image

Without:
image



And probably a more obvious example of it being hard to see:
With:
image

Without:
image

**Comment by J0nathan550** [comment]
I don't like purple for launcher that uses black-white theme. Better to make the accent color white. We will able to see everything.

**Comment by J0nathan550** [comment]
> I don't like purple for launcher that uses black-white theme. Better to make the accent color white. We will able to see everything.

SIT Manager 04 03 2024 14_10_48

**Comment by Plootie** [comment]
Finally got myself a change to properly look over this now that the linux stuff is resolved (mostly). Despite all my effort I believe you are correct that there is absolutely no way to access those properties on the xaml side which is truly horrifying to me. I did some experimenting and found `#7f7f7f` seemed to be a good intermediate between the two for now. It would be interesting if we could allow users to edit this themselves through the settings menu at some point

**Comment by J0nathan550** [comment]
> 7f7f7f

this color is awesome let me push again

**Pull request merged**